### PR TITLE
Improve Zweihander layout for the Ergodox EZ

### PR DESCRIPTION
**Pull request by adiabatic**
## Description

This updates the Zweihander layout with a handy backtick key, line-mover keys for Visual Studio Code, and fiddles with the insidemost bottom four thumb buttons.

It also restores the `config.h` file that turns down the power consumption so it can be used on iPads.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

(none)

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

> `<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->`

Do my `keymap.c` and `config.h` need GPL-compatible license headers now? For this sort of thing, I'd prefer a copycenter license like the MIT or zlib license.

### CODE REVIEW

**Review by drashna — APPROVED** [review]

**Comment by adiabatic** [comment]
I added some functionality just now.

I'm not going outside of my layout to do so, but this is my first time using `process_record_user()`, even though everything seems OK after copy-pasting the relevant code from <https://docs.qmk.fm/#/feature_macros?id=using-macros-in-c-keymaps>.

File: layouts/community/ergodox/zweihander-macos/keymap.c [hunk 1]
```diff
@@ -8,6 +8,24 @@
...
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+    case NEWITEM:
+        if (record->event.pressed) {
+            // when keycode NEWITEM is pressed
+            SEND_STRING("\n- "); // starts a new item in a Markdown-style list
+        } else {
+            // when keycode NEWITEM is released
```
Comment (drashna) [inline layouts/community/ergodox/zweihander-macos/keymap.c:22]: You don't need this part, and can be removed. 
```suggestion
```

Comment (adiabatic) [reply layouts/community/ergodox/zweihander-macos/keymap.c]: I can understand why people would want to remove a do-nothing `else` clause, but I hate to remove it entirely without a trace. I've removed it in [7dd0c4f](https://github.com/qmk/qmk_firmware/pull/18737/commits/7dd0c4f7995bb724b6879d9f47e0a068728eddbc), but left in a comment explaining what an `else` clause would be useful for. What do you think of it now?

**Comment by adiabatic** [comment]
@drashna I fixed the merge conflict. I don't know why it had one, but somehow it did.

Is this OK to merge in now?

**Review by fauxpark — APPROVED** [review]

**Pull request merged**